### ISSUE TITLE
Proposal rejection related

### ISSUE THREAD

**Issue by Ayush23Dash** [OrkoHunter/gsoc-FAQs#12]
NOTE:
 - Have you read [How to Ask Questions the Smart Way](http://www.catb.org/esr/faqs/smart-questions.html) ?
 - Please fill out the template below to get a definite answer.

## What is your query? 

> [I am currently working on a proposal by Mozilla and have a doubt.My mentor told me that it is still not clear whether the proposal will be accepted or not
So I want to ask that is there something like that even if an organisation is selected even then some of their proposals are rejected?]

## Is this question specific to a participating organization?

> Y

## Have you read the [FLOSS Manual](http://write.flossmanuals.net/gsocstudentguide) yet?

> N. Tip: Be proactive.

## Have you read the questions in the [README](https://github.com/OrkoHunter/gsoc-FAQs/blob/master/README.md)?

> Y

## Did you Google search this before posting?

> [LMGTFY](https://www.google.com)

## Do you think this question will help other applicants?

> Y

**Comment by thealphadollar** [comment]
Yes. An org may choose to not accept your proposal even when they selected as an organization in GSoC.
